files/default/neutron-ha-tool.py

```diff
@@ -922,6 +925,41 @@ def pick(self):
         return random.choice(self.agents)
 
 
+class LeastBusyAgentPicker(object):
+    def __init__(self, qclient, agents):
+        self.now = datetime.datetime.now
```

Comment:
The intention of having now being injectable was because I wanted to have the ability to replace `now` with a mock should we need. Looking at the tests, this was never used, so it's a candidate for YAGNI now. At a later change this has been removed also. I might move that change here, so we don't do the add-and-remove cycle.